docs/coding.rst

```diff
@@ -667,20 +672,17 @@ To use these macros in files, we simply import a macro to the page's HTML code a
 
     // bedrock/bedrock/firefox/templates/firefox/{specific-page}.html
 
-    {% from "macros-protocol.html" import split, picto with context %}
+    {% from "macros-protocol.html" import billboard with context %}
 
-    {% call split(
-        image_url='img/firefox/browsers/hero.jpg',
-        include_highres_image=True,
-        block_class='mzp-l-split-center-on-sm-md',
-        media_class='mzp-l-split-media-overflow',
-        media_after=True
-    ) %}
-        <h1>This is Mozilla</h1>
-        <p>Get the privacy you deserve with Firefox</p>
-    {% endcall %}
+    {{ billboard(
+        title='This is Firefox.',
+        ga_title='This is Firefox',
+        desc='Firefox is an awesome web browser.',
+        link_cta='Click here to install',
+        link_url='mozilla.org/firefox',
+      )}}
 
-Because Split styles are not global, we still have to import the page-specific Protocol styles in SCSS:
+Because component styles are not global, we still have to import the page-specific Protocol styles in SCSS:
```

Comment:
Nit: I think it reads better as "Because not all component styles are global" (there's probably a name for this rule in English grammar but I don't know it)